src/components/performances/PerformanceForm.jsx

```diff
@@ -17,7 +18,7 @@ class PerformanceForm extends React.Component {
     const { currentValues } = props;
 
     this.state = {
-      danceEntry: currentValues.danceEntry || 0,
+      danceEntry: currentValues.danceEntry || '',
```

Comment:
Hm, if this danceEntry is actually the integer value we input on creation, shouldnt we keep it's default state an integer value, or is this just to hide the value if it doesn't exist?